### DISCUSSION TITLE
Lighthouse use 28 Day Trailing Data Like Page Speed Insights

### DISCUSSION THREAD

**Discussion by kellyjjrusk** [GoogleChrome/lighthouse#15615] [Q&A] [answered]
I have a site with two URLs. 1 was a test URL before we switched over the Production URL (old site) that was already in use. When I run Lighthouse on the Test URL it has a much higher score and information that seems tied to our features... but the Production URL that was used on an existing site (we replaced the site with a better version) shows lower scores and information that seem to be tied to the old site.

In essence, both our of URLs go to the same backend... but the Production URL shows much different scoring... Does this mean that Lighthouse has 28 day trailing data like PageSpeed Insights so that the old site data is still part of when I run a report? My goal is to release improvements based on the Lighthouse reports and retest... without the old site data mixed in.

**Comment by adamraine** [comment]
Lighthouse does not use any field data, it is a lab tool that analyzes a single page session. If you run Lighthouse on your production URL today, it will analyze the production page as it appears today.

For variance in performance score, please see https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md

**Reply by kellyjjrusk** [reply]
Thank you. That is what I thought, but appreciate the fast confirmation. Suspect the CDN (its tied per URL) has cached data that needs to be purged per the @connorjclark  reply

**Comment by connorjclark** [comment, marked as answer]
It does not.

What you may be seeing is that your CDN is caching, which on a per-region basis may lag behind what you see when you hit your site from different locations. Just a wild guess.

Are the two versions of the site visually distinct? If so, see the page thumbnail to confirm what version PSI got.

**Comment by kellyjjrusk** [comment]
Thank you all. I suspect the CDN is still updating. The old and new sites look/feel are identical. I have done multiple runs on the test URL and the Prod URL. Both are behind the same CDN. The test URL is always higher (ex. Best Practices in the 90s vs. 60s on the Prod URL)
